packages/block-editor/src/components/use-block-drop-zone/index.js

```diff
@@ -260,7 +260,10 @@ export default function useBlockDropZone( {
 				return;
 			}
 
-			const sourceBlockIndex = getBlockIndex( sourceClientIds[ 0 ] );
+			const sourceBlockIndex = getBlockIndex(
+				sourceClientIds[ 0 ],
+				sourceRootClientId
```

Comment:
Why does `getBlockIndex( clientId )` require a second parameter if client IDs are supposed to be unique?